### ISSUE TITLE
The size of your trained model

### ISSUE THREAD

**Issue by myaolee** [fab-jul/imgcomp-cvpr#14]
@fab-jul 
hello~The Model-size of ckpts provied by you(0515_1103) is 136530KB = 1KB(checkpoint) +117723KB(.data) + 32KB(.index) + 18687KB(.meta) + 60KB(.pkl).
But the size of the model I trained with the network you provided is 232814 KB = 1KB(checkpoint) + 196160KB(.data) + 47KB(.index) + 36511KB(.meta) + 95KB(.pkl).

I think the model-size trained on the same network should be the same.  So what i am confused about is whether the ckpts you provided are trained by the network you gave here?  What is the difference between the network you trained and the network provided here?
I am looking forward to your reply.

**Comment by fab-jul** [comment]
The checkpoints provided are trained with the exact code provided.

I do not know where the difference in size comes from, might be due to different TensorFlow/Python/OS version? Did you find anything out?

**Comment by madisa1** [comment]
@fab-jul  I've had the same issue but it is not the model size that got me confused but the results I got when I trained the same network on ImageNet. The main difference between your training and mine is that you used ImageMagick to crop the images to 256 and I used cv2 which i guess is not a problem.
Still, the quality of the image is different. So that got me confused, specially when in the paper you mentiond different values for beta (10 in the paper and 500 in your code) and the loss's factor (k_mssim=100 in the paper and 5000 in your code).

Thank you.
